app/components/common/TableCardBody.tsx

```diff
@@ -1,9 +1,24 @@
+import { cva, VariantProps } from 'class-variance-authority';
 import React from 'react';
 
-export function TableCardBody({ children }: { children: React.ReactNode }) {
+const tableVariants = cva(["table table-sm card-table"], {
+    defaultVariants: {
+        layout: "compact"
+    },
+    variants: {
+        layout: {
+            "compact": ["table-nowrap"],
+            "expanded": []
+        }
+    }
+})
+
+export interface TableCardBodyProps extends VariantProps<typeof tableVariants>, React.PropsWithChildren {}
+
+export function TableCardBody({ children, ...props }: TableCardBodyProps) {
```

Comment:
What is this doing?

Reply:
That code add "variants" for the table. Original one contains `table-nowrap` class that adds `no-wrap` to all the cells.
For the feature we do not need this, as description might be quite long and with `nowrap` there will be a scroll. That's not very convenient.

That's why I've added additional variant (`expanded`), which allows to not use `no-wrap` and use it for all the needed cells.

All the previous tables won't be affected